### ISSUE TITLE
Nothing works

### ISSUE THREAD

**Issue by krischal111** [foresterre/cargo-features#1]
It doesn't list all the features of a crate. It just links to this website.

**Comment by Yevgnen** [comment]
Yes, what does that mean?

**Comment by krischal111** [comment]
This means, 3 months ago, when I used this crate, it didn't did the job it was supposed to do and just linked to this GitHub which was not the thing expected by that crate, at least for me. That's all.

**Comment by Yevgnen** [comment]
Yes...I was agreeing with you and wondering what does that `See current development version at: https://github.com/foresterre/cargo-features` mean.

**Comment by krischal111** [comment]
I presume we could pull the crate from the GitHub instead of cargo to make it run. I don't know if that's the case, I haven't used rust for about 3 months now, so things might have changed.

**Comment by computermouth** [comment]
probably squatting on this name, knowing that people will accidentally install this instead of `cargo-feature`, and probably run some malware scheme or sell it to someone who wants to

**Comment by paulirotta** [comment]
You probably want [cargo-features-manager](https://crates.io/crates/cargo-features-manager)
